### PR TITLE
News article images - make image fit tile, add tests

### PR DESCRIPTION
**Pull request by DamienBurkeWN**
### Testing

- `yarn test NewsArticleFeaturedBlock`
- Look at the news article featured block stories and verify wide/tall images aren't cropped now
- Look at the news landing page example and verify wide/tall images aren't cropped now

### CODE REVIEW

File: src/library/structure/NewsArticleFeaturedBlock/NewsArticleFeaturedBlock.storydata.js [hunk 1]
```diff
@@ -7,31 +7,30 @@ export const NewsArticleData = [
     "url": "/",
     "title": "Veniam nostrud excepteur elit non cupidatat deserunt sunt amet ad ipsum sunt incididunt.",
     "date": 1614178638,
-    "image720x405": "https://core-cms.bfi.org.uk/sites/default/files/styles/responsive/public/1440/810/1/2020-08/bfi-film-academy-2020-recruitment-campaign-watershed-two-girls-behind-camera.jpeg",
-    "image72x41": "https://core-cms.bfi.org.uk/sites/default/files/styles/responsive/public/144/81/1/2020-08/bfi-film-academy-2020-recruitment-campaign-watershed-two-girls-behind-camera.jpeg",
+    "image720x405": "https://cms-staging.westnorthants.gov.uk/sites/default/files/styles/responsive/public/720/405/0/2022-02/HMD-Facebook-graphic_3.jpg",
```
Comment (DamienBurkeWN) [inline src/library/structure/NewsArticleFeaturedBlock/NewsArticleFeaturedBlock.storydata.js:10]: Different images to demo the change

File: src/library/structure/NewsArticleFeaturedBlock/NewsArticleFeaturedBlock.storydata.js [hunk 2]
```diff
@@ -7,31 +7,30 @@ export const NewsArticleData = [
...
     "date": 1614178638,
-    "image720x405": "https://core-cms.bfi.org.uk/sites/default/files/styles/responsive/public/1440/810/1/2020-08/bfi-film-academy-2020-recruitment-campaign-watershed-two-girls-behind-camera.jpeg",
-    "image72x41": "https://core-cms.bfi.org.uk/sites/default/files/styles/responsive/public/144/81/1/2020-08/bfi-film-academy-2020-recruitment-campaign-watershed-two-girls-behind-camera.jpeg",
+    "image720x405": "https://cms-staging.westnorthants.gov.uk/sites/default/files/styles/responsive/public/720/405/0/2022-01/For%20The%20Love%20Of%20Books%20-%20logo%20-%20landscape.png",
+    "image72x41": "https://cms-staging.westnorthants.gov.uk/sites/default/files/styles/responsive/public/72/41/0/2022-01/For%20The%20Love%20Of%20Books%20-%20logo%20-%20landscape.png",
     "imageAltText": "The alt text"
   },
   {
-    "id": "6036694ed3c1de9114024a02",
+    "id": "6036694ed3c1de9114024a03",
```
Comment (DamienBurkeWN) [inline src/library/structure/NewsArticleFeaturedBlock/NewsArticleFeaturedBlock.storydata.js:33]: Fixed these non-unique IDs too

File: src/library/structure/NewsArticleFeaturedBlock/NewsArticleFeaturedBlock.styles.js [hunk 3]
```diff
@@ -1,101 +1,105 @@
-import styled from "styled-components";
+import styled from 'styled-components';
```
Comment (DamienBurkeWN) [inline src/library/structure/NewsArticleFeaturedBlock/NewsArticleFeaturedBlock.styles.js:1]: Mostly prettier...

File: src/library/structure/NewsArticleFeaturedBlock/NewsArticleFeaturedBlock.styles.js [hunk 4]
```diff
@@ -105,62 +109,60 @@ export const ArticleContainer = styled.a`
...
 
 export const ImageContainer = styled.span`
   display: block;
   width: 100%;
   height: ${imageHeightMobile}px;
   overflow: hidden;
-  background-image: url("${props => props.background}");
-  background-size: cover;
+  background-image: url('${(props) => props.background}');
+  background-size: contain;
```
Comment (DamienBurkeWN) [inline src/library/structure/NewsArticleFeaturedBlock/NewsArticleFeaturedBlock.styles.js:141]: This and the next line is the actual change

File: src/library/structure/NewsArticleFeaturedBlock/NewsArticleFeaturedBlock.tsx [hunk 5]
```diff
@@ -1,55 +1,62 @@
...
-                </Styles.ViewAllContainer>
-            }
-            </>
-        )
-    } else {
-        return (
-            <></>
-        )
-    }
+/**
```
Comment (DamienBurkeWN) [inline src/library/structure/NewsArticleFeaturedBlock/NewsArticleFeaturedBlock.tsx:9]: Just prettier plus adding this comment

File: src/library/structure/NewsArticleFeaturedBlock/NewsArticleFeaturedBlock.types.ts [hunk 6]
```diff
@@ -1,20 +1,20 @@
-
 export interface NewsArticleFeaturedBlockProps {
   /**
-   * Array of results
+   * Array of news article data
```
Comment (DamienBurkeWN) [inline src/library/structure/NewsArticleFeaturedBlock/NewsArticleFeaturedBlock.types.ts:3]: Just prettier and comment alterations

**Comment by chrisrhymes** [comment]
Looks good, just confused about the mock up in Trello which has the image on the right instead of above. Does the same change need to be done on the News Article List too?

File: src/library/pages/News/News.tsx [hunk 7]
```diff
@@ -1,96 +1,117 @@
-
-import React from "react";
```
Comment (DamienBurkeWN) [inline src/library/pages/News/News.tsx:2]: This file just prettified and a small bug fixed

File: src/library/pages/News/News.tsx [hunk 8]
```diff
@@ -1,96 +1,117 @@
...
+              sections={[
+                {
+                  title: NewsArticleFilterFields.search.title,
+                  content: (
+                    <>
+                      <PageStructures.Searchbar
+                        id="news-search"
+                        searchTerm={newsArticleData.searchTerm ? newsArticleData.searchTerm : undefined}
+                        isLight={true}
+                        submitInfo={{
```
Comment (DamienBurkeWN) [inline src/library/pages/News/News.tsx:50]: Removed the [] wrapping to fix the type error

File: src/library/structure/NewsArticleList/NewsArticleList.stories.tsx [hunk 9]
```diff
@@ -1,37 +1,34 @@
...
-import React from "react";
+import React from 'react';
 import { Story } from '@storybook/react/types-6-0';
-import NewsArticleList from "./NewsArticleList";
-import { NewsArticleListProps } from "./NewsArticleList.types";
+import NewsArticleList from './NewsArticleList';
+import { NewsArticleListProps } from './NewsArticleList.types';
 import { SBPadding } from '../../../../.storybook/SBPadding';
-import { newsArticleData } from './NewsArticleData';
+import { newsArticleData } from './NewsArticleList.storydata';
```
Comment (DamienBurkeWN) [inline src/library/structure/NewsArticleList/NewsArticleList.stories.tsx:6]: Mostly prettier and picking up changed storydata file

File: src/library/structure/NewsArticleList/NewsArticleList.storydata.js [hunk 10]
```diff
@@ -0,0 +1,169 @@
+import Uri from 'jsuri';
```
Comment (DamienBurkeWN) [inline src/library/structure/NewsArticleList/NewsArticleList.storydata.js:1]: Prettier version of file previously named NewsArticleData.js

File: src/library/structure/NewsArticleList/NewsArticleList.styles.js [hunk 11]
```diff
@@ -1,115 +1,108 @@
...
 export const ImageContainer = styled.span`
   display: block;
   width: 100%;
-  height: ${imageHeightMobile}px;
+  height: 150px;
   overflow: hidden;
-  background-image: url("${props => props.background}");
-  background-size: cover;
+  background-image: url('${(props) => props.background}');
+  background-size: contain;
```
Comment (DamienBurkeWN) [inline src/library/structure/NewsArticleList/NewsArticleList.styles.js:89]: Actual change this and next line

File: src/library/structure/NewsArticleList/NewsArticleList.tsx [hunk 12]
```diff
@@ -1,49 +1,55 @@
+import React from 'react';
```
Comment (DamienBurkeWN) [inline src/library/structure/NewsArticleList/NewsArticleList.tsx:1]: Just prettier

File: src/library/structure/NewsArticleList/NewsArticleList.stories.tsx [hunk 13]
```diff
@@ -1,37 +1,34 @@
...
 import { newsArticleData } from './NewsArticleData';
 
 export default {
-    title: 'Library/Components/News Article List',
-    component: NewsArticleList,
-    parameters: {
-      status: {
-        type: 'stable', // 'beta' | 'stable' | 'deprecated' | 'releaseCandidate'
-      }
+  title: 'Library/Components/News Article List',
```
Comment (chrisrhymes) [inline src/library/structure/NewsArticleList/NewsArticleList.stories.tsx:9]: Should this be moved from Components to Structure?

**Review by chrisrhymes — APPROVED** [review]
Looks good, just the query about the title to move to Structure. Took me a while to find it in storybook.

**Comment by DamienBurkeWN** [comment]
Good spot, sorted

**Pull request merged**